generators/php/sdk/src/endpoint/request/WrappedEndpointRequestGenerator.ts

```diff
@@ -0,0 +1,113 @@
+import { php, PhpFile, FileGenerator, FileLocation } from "@fern-api/php-codegen";
+import { join, RelativeFilePath } from "@fern-api/fs-utils";
+import { HttpEndpoint, SdkRequestWrapper, ServiceId } from "@fern-fern/ir-sdk/api";
+import { SdkCustomConfigSchema } from "../../SdkCustomConfig";
+import { SdkGeneratorContext } from "../../SdkGeneratorContext";
+
+export declare namespace WrappedEndpointRequestGenerator {
+    export interface Args {
+        serviceId: ServiceId;
+        wrapper: SdkRequestWrapper;
+        context: SdkGeneratorContext;
+        endpoint: HttpEndpoint;
+    }
+}
+
+export class WrappedEndpointRequestGenerator extends FileGenerator<
+    PhpFile,
+    SdkCustomConfigSchema,
+    SdkGeneratorContext
+> {
+    private wrapper: SdkRequestWrapper;
+    private serviceId: ServiceId;
+    private endpoint: HttpEndpoint;
+    private classReference: php.ClassReference;
+    private location: FileLocation;
+
+    public constructor({ wrapper, context, serviceId, endpoint }: WrappedEndpointRequestGenerator.Args) {
+        super(context);
+        this.wrapper = wrapper;
+        this.serviceId = serviceId;
+        this.endpoint = endpoint;
+        this.classReference = this.context.getRequestWrapperReference(this.serviceId, this.wrapper.wrapperName);
+        this.location = this.context.getLocationForWrappedRequest(this.serviceId);
+    }
+
+    protected doGenerate(): PhpFile {
+        const class_ = php.class_({
+            ...this.classReference
+        });
```

Comment:
this is unimportant and more of a curiosity -- why not do this?
```suggestion
        const class_ = php.class_(this.classReference);
```

Reply:
Good catch, will commit your suggestion!